mytoyota/models/trip.py

```diff
@@ -0,0 +1,85 @@
+"""Models for trips and detailed trips."""
+from __future__ import annotations
+
+class TripEvent:
+
+    @property
+    def latitude(self) -> float:
+        """Event latitude."""
+        return float(self._data.get("lat", 0.0))
```

Comment:
Where is `_data` defined? It seems that a constructor is missing that should be defining and initializing `_data`.

Reply:
Good catch, I should have subclassed the base data class, updated in https://github.com/DurgNomis-drol/mytoyota/pull/188/commits/ea43d7a43bff9c20f0c3d93650efab802ed4d6d7

Reply:
Subclassing the base data class, is indeed the perfect solution. 👍